src/Illuminate/Console/Scheduling/CallbackEvent.php

```diff
@@ -59,13 +59,11 @@ public function run(Container $container)
 		try {
 			$response = $container->call($this->callback, $this->parameters);
 		} catch (Exception $e) {
-			$this->removeMutex();
-
 			throw $e;
+		} finally {
```

Comment:
Does PHP require having a catch if you are just gonna throw it? Can you just have a try and finally?